Sources/VSM/Documentation.docc/Quickstart/QuickstartGuide.tutorial

```diff
@@ -0,0 +1,351 @@
+            }
+            
+            @Step {
+                You'll need an "initialized" state to kick off the loading action. Define a case for this state and its associated model value.
+                
+                @Code(name: "BlogEntryViewState.swift", file: quickstart-state-loader-state)
+            }
+            
+            @Step {
+                Fill in the "loader" model by adding the `loadEntry()` action which returns a series of view states as a Combine publisher.
+                
+                > Note: To learn more about VSM action types, see <doc:ModelActions>.
+                
+                @Code(name: "BlogEntryViewState.swift", file: quickstart-state-loader-model)
+            }
+            
+            @Step {
+                Now, define a "loading" state which should be immediately returned when the blog entry starts loading. Create an optional error model for use if the blog entry fails to load.
+                
+                > Note: Not all view states need an associated model. For more information on how building the state in VSM, see <doc:StateDefinition>.
+                
+                
+                @Code(name: "BlogEntryViewState.swift", file: quickstart-state-loading-state)
+            }
+            
+            @Step {
+                Fill in the error model definition by adding the error message that you want the user to see, and a retry action that returns a series of view states.
+                
+                @Code(name: "BlogEntryViewState.swift", file: quickstart-state-error-model)
+            }
+            
+            @Step {
+                Finally, create the loaded model that contains the blog entry information that you'd like to display.
+                
+                @Code(name: "BlogEntryViewState.swift", file: quickstart-state-loaded-model)
+            }
+        }
+    }
+    
+    @Section(title: "Building the View") {
+        @ContentAndMedia {
+            Now that we've defined our Feature Shape, we can start building the view. We will utilize two VSM framework types to make it easier to follow the pattern: <doc:ViewStateRendering> and <doc:StateContainer>.
+            
+            > Note: This example covers only SwiftUI, but UIKit is fully supported by VSM. To learn more about building views in VSM, see <doc:ViewDefinition-SwiftUI> or <doc:ViewDefinition-UIKit>.
+            
+            @Image(source: quickstart-view.jpg, alt: "A depiction of the view interacting with the state")
+        }
+        
+        @Steps {
+            @Step {
+                First, start off by declaring your SwiftUI view.
+                
+                @Code(name: "BlogEntryView.swift", file: quickstart-view-start)
+            }
+            
+            @Step {
+                Next, conform your view to the `ViewStateRendering` protocol. You are required add a `StateContainer` property. 
```

Comment:
Hey @albertbori 👋 
**_Question:_** What do you think about adding a note as to why the StateContainer property is important?